idna/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "idna"
-version = "0.4.0"
+version = "0.5.0"
```

Comment:
It's pretty unclear to me why this got a semver-incompatible bump? I don't see any changes that require it -- causing me to have duplicate dependencies where none should be needed.